tests/playwright-test/snapshot-path-template.spec.ts

```diff
@@ -79,6 +79,9 @@ test('tokens should expand property', async ({ runInlineTest }, testInfo) => {
     }, {
       name: 'testFileName',
       snapshotPathTemplate: '{testFileName}',
+    }, {
+      name: 'testFileNameWithoutExtension',
+      snapshotPathTemplate: '{testFileNameWithoutExtension}',
```

Comment:
The actual expects need to be extended too to cover various use cases of the new token.